bluebottle/members/tests/test_admin.py

```diff
@@ -301,13 +301,15 @@ class MemberPlatformSettingsAdminTestCase(BluebottleAdminTestCase):
 
     def test_require_location(self):
         self.app.set_user(self.superuser)
+        user = BlueBottleUserFactory.create()
         page = self.app.get(reverse('admin:members_memberplatformsettings_change'))
         form = page.forms[0]
         form['require_office'].checked = True
 
         form.submit()
         settings_platform = MemberPlatformSettings.load()
-        self.assertTrue(settings_platform.require_office)
+        if user.place is None:
+            self.assertTrue(settings_platform.require_office)
 
```

Comment:
Why would you expect that `settings_platform.require_office` would be something different if a certain user would exist? That is not at all what this test was supposed to verify. If this would have been the case, something would be really wrong.